tests/contrib/loguru/test_loguru_logging.py

```diff
@@ -0,0 +1,225 @@
+import json
+
+from loguru import logger
+import pytest
+
+from ddtrace import config
+from ddtrace import tracer
+from ddtrace.constants import ENV_KEY
+from ddtrace.constants import SERVICE_KEY
+from ddtrace.constants import VERSION_KEY
+from ddtrace.contrib.loguru import patch
+from ddtrace.contrib.loguru import unpatch
+from tests.utils import override_global_config
+
+
+captured_logs = []
+
+
+def _test_logging(output, span, env, service, version):
+    dd_trace_id, dd_span_id = (span.trace_id, span.span_id) if span else (0, 0)
+
+    assert "Hello" in json.loads(output[0])["text"]
+    assert json.loads(output[0])["dd.trace_id"] == str(dd_trace_id)
+    assert json.loads(output[0])["dd.span_id"] == str(dd_span_id)
+    assert json.loads(output[0])["dd.env"] == env or ""
+    assert json.loads(output[0])["dd.service"] == service or ""
+    assert json.loads(output[0])["dd.version"] == version or ""
+
+    output.clear()
+
+
+@pytest.fixture(autouse=True)
+def patch_loguru():
+    try:
+        patch()
+        logger.add(captured_logs.append, serialize=True)
+        yield
+    finally:
+        unpatch()
+
+
+@pytest.fixture(autouse=True)
+def global_config():
+    with override_global_config({"service": "logging", "env": "global.env", "version": "global.version"}):
+        yield
+
+
+def test_log_trace_global_values():
+    """
+    Check trace info includes global values over local span values
```

Comment:
Why wouldn't we want local span values to override? If a span has a different service name than the one set with DD_SERVICE, won't we want it to show up as a separate service in both logs and APM, not just APM?

Reply:
This follows the practice of what the current `logging` [integration](https://github.com/DataDog/dd-trace-py/blob/272159336928a7457e650ff79635c9801368af03/tests/contrib/logging/test_logging.py#L217) does. Most likely, this is to ensure that all logs stay within a certain configuration instead of being scattered all over the place since one application would have one logger attached to it.

Reply:
Got it, thought that was probably it. Thanks for confirmation.